crates/next-core/src/next_font_google/mod.rs

```diff
@@ -3,6 +3,7 @@ use indexmap::IndexMap;
 use indoc::formatdoc;
 use once_cell::sync::Lazy;
 use turbo_tasks::primitives::{OptionStringVc, OptionU16Vc, StringVc, U32Vc};
+#[allow(unused_imports, /* reason = "this is used in tests" */)]
```

Comment:
```suggestion
#[cfg(debug_assertions)]
```

But best just move the `use ...` into the block below.